runelite-client/src/main/java/net/runelite/client/plugins/bankvalue/BankValueConfig.java

```diff
@@ -30,26 +30,48 @@
 import net.runelite.client.config.ConfigItem;
 
 @ConfigGroup(
-	keyName = "bankvalue",
-	name = "Bank Value",
-	description = "Shows the value of your bank and/or current tab")
+		keyName = "bankvalue",
+		name = "Bank Value",
+		description = "Shows the value of your bank and/or current tab")
```

Comment:
Why was the indent on this changed? If IDEA is doing it automatically you should review the following article:
https://github.com/runelite/runelite/wiki/Checkstyle-with-IntelliJ-IDEA